Documentation/GettingStartedWithTheMRTK.md

```diff
@@ -0,0 +1,139 @@
+
+Configuring your scene is extremely simple by simply selecting the following from the Editor menu:
+> Mixed Reality Toolkit -> Configure
+
+![](/External/ReadMeImages/MRTK_ConfigureScene.png)
+
+Once this completes, you will see the following in your Scene hierarchy:
+
+![](/External/ReadMeImages/MRTK_SceneSetup.png)
+
+Which contains the following:
+
+* Mixed Reality Toolkit - The toolkit itself, providing the central configuration and activation point for the entire framework.
+* MixedRealityPlayspace - The parent object for the headset, which ensures the headset / controllers and other required systems are managed correctly in the scene.
+* The Main Camera is moved as a child to the Playspace - Which allows the playspace to manage the camera in conjunction with the SDK's
+* UIRaycastCamera added as a child to the Main Camera - To enable seamless UI interactions through the toolkit
+
+> **Note** While working in your scene, **DON'T move the Main Camera** (or the playspace) from the scene origin (0,0,0).  This is controlled by the MRTK and the active SDK.
+> If you need to move the players start point, then **move the scene content and NOT the camera**!
+
+### 4. Hit play
+
+You are now ready to start building your Mixed Reality Solution, just start adding content and get building.
+Switch to other platforms (ensure they have XR enabled in their player settings) and your project will still run as expected without change.
+
+## Configuring your project
+
+The Mixed Reality Toolkit configuration is all centralized on one place and attached to the MixedRealityToolkit object in your active scene.
+
+![](/External/ReadMeImages/MRTK_ActiveConfiguration.png)
+
+Clicking on this profile will show the configuration screens for the Mixed Reality Toolkit:
+
+![](/External/ReadMeImages/MRTK_MixedRealityToolkitConfigurationScreen.png)
+
+From here you can navigate to all the configuration profiles for the MRTK, including:
+
+* Main Mixed Reality Toolkit Configuration
+* Camera Settings
+* Input System Settings
+* Boundary Settings
+* Teleporting Settings
+* Spatial Awareness Settings
+* Diagnostics Settings
+* Additional Services Settings
+* Input Actions Settings
+* Input Actions Rules
+* Pointer Configuration
+* Gestures Configuration
+* Speech Commands
+* Controller Mapping Configuration
+* Controller Visualization Settings
+
+As you can see there are lots of options available and more will come available as we progress through the beta.
+
+When you start a new project, we provide a default set of configurations with every option turned on, styled for a fully cross platform project.  These defaults are "Locked" to ensure you always have a common start point for your project and we encourage you to start defining your own settings as your project evolves.  For this we provide options to either:
+
+* Copy the defaults in to a new profile for you to start customizing it for your project
```

Comment:
Maybe mention where the new profiles are generated to. Or a short description of all folders.